qwik-graphql-tailwind/src/components/pagination/pagination.tsx

```diff
@@ -24,29 +22,21 @@ export const Pagination = component$(({ pageInfo, owner }: PaginationProps) => {
   });
 
   return (
-    <div className={styles.container}>
-      <span className={styles.group}>
+    <div className="flex items-center justify-center mt-4">
+      <span className="relative z-0 inline-flex shadow-sm rounded-md">
         <button
           type="button"
           disabled={!pageInfo.hasPreviousPage || !pageInfo.startCursor}
           onClick$={handlePreviousClick$}
-          className={
-            !pageInfo.hasPreviousPage || !pageInfo.startCursor
-              ? cn(styles.button, styles.buttonPrev)
-              : cn(styles.button, styles.buttonPrev)
-          }
```

Comment:
Seems that this ternary is useless, the class is always the same. Don't know if you have some insight I haven't. 
lmk